scripts/darkCapture.sh

```diff
@@ -2,36 +2,43 @@
 
 # This file is "source"d into another.
 # "${CURRENT_IMAGE}" is the name of the current image we're working on and is passed to us.
+ME2="$(basename "${BASH_SOURCE[0]}")"
```

Comment:
ME, ME2,.... is a quick and dirty solution. i think bash supports local variables....

Reply:
Since darkCapture.sh is "source"d into another file, any variable set in darkCapture.sh will overwrite what was set in the invoking script, as you noticed and commented on in a Discussion item.
I'm not sure what a local variable is, but if darkCapture.sh used $ME and didn't overwrite what the invoker set, I think it would be confusing.